client.js

```diff
@@ -115,7 +115,7 @@ class FilesCollection extends FilesCollectionCore {
     if (!config.disableSetTokenCookie) {
       const setTokenCookie = () => {
         if (Meteor.connection._lastSessionId) {
-          cookie.set('x_mtok', Meteor.connection._lastSessionId, { path: '/', sameSite: 'Lax', secure: true, httpOnly: true });
+          cookie.set('x_mtok', Meteor.connection._lastSessionId, { path: '/', sameSite: 'Lax', secure: true });
```

Comment:
Are you sure this is not just a local testing issue? Are you using `localhost`? Would the following be more appropriate:

```suggestion
          cookie.set('x_mtok', Meteor.connection._lastSessionId, { path: '/', sameSite: 'Lax', secure: Meteor.isProduction, httpOnly: Meteor.isProduction });
```